docs/advanced/naifu.md

```diff
@@ -0,0 +1,110 @@
+# 部署 NAIFU
+
+NAIFU 是匿名网友打包的一套 [NovelAI](https://novelai.net/) 的前端与后端应用，通过它可以快速搭建一个第三方的 NovelAI 服务。
+
+:::tip
+由于 NAIFU 在传输过程中被多次修改，产生了许多不同的版本，各版本之间的部署方式可能不尽相同。本文档仅针对最初发布的版本进行说明。
+如果你发现了本文档对部署方式的描述与你所下载的 NAIFU 附带的说明文件不相符，请以你下载的版本为准。
+:::
+
+## 下载并解压 NAIFU
+
+透过磁力链接或网盘等方式下载 NAIFU，然后解压到一个目录中。
+
+如果你下载的内容中包含了 program.zip 文件，请将其解压到当前目录，不需要解压到子目录中。
+
+最终的目录结构应该如下所示：
+
+```text
+naifu/
+ |- clip/
+ |- hydra_node/
+ |- k_diffusion/
+ |- ldm/
+ |- main.py
+ |- models/
+ |- README.txt
+ |- requirements.txt
+ |- run.bat
+ |- run.sh
+ |- setup.bat
+ |- setup.sh
+ |- static/
+ |- taming/
+```
+
+## 安装（Windows）
```

Comment:
```suggestion
## 安装 (Windows)
```

![image](https://user-images.githubusercontent.com/33423008/197457399-f0d7e658-ea1a-478c-bfe9-3fc3d30e89fd.png)

Reply:
You mean all full-width brackets should be changed to half-width ones?

Reply:
yes